projects/gnutls/build.sh

```diff
@@ -15,8 +15,15 @@
 #
 ################################################################################
 
+CONFIGURE_FLAGS=""
+if [[ $SANITIZER_FLAGS = "*sanitize=memory*" ]]
```

Comment:
I believe that it would be better to examine `$CFLAGS` here, as we guarantee to set it as per [the docs](https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md#build-support), while `$SANITIZER_FLAGS` is an internal stuff from infra/ and may be renamed/removed unexpectedly.